### PR TITLE
[prometheus-node-exporter] be able to set environment variables

### PR DESCRIPTION
**Pull request by TomasKohout**
Signed-off-by: Tomas Kohout <tomas.kohout1995@gmail.com>

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-node-exporter/values.yaml [hunk 1]
```diff
@@ -76,7 +82,8 @@ securityContext:
   runAsNonRoot: true
   runAsUser: 65534
 
-containerSecurityContext: {}
+containerSecurityContext:
```
Comment (monotek) [inline charts/prometheus-node-exporter/values.yaml:85]: unrelated change. please keep as it is.

File: charts/prometheus-node-exporter/values.yaml [hunk 2]
```diff
@@ -48,7 +53,8 @@ updateStrategy:
   rollingUpdate:
     maxUnavailable: 1
 
-resources: {}
+resources:
```
Comment (monotek) [inline charts/prometheus-node-exporter/values.yaml:56]: unrelated change. please keep as it is.

File: charts/prometheus-node-exporter/Chart.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@ keywords:
   - prometheus
   - exporter
 type: application
-version: 3.0.2
+version: 3.0.3
```
Comment (monotek) [inline charts/prometheus-node-exporter/Chart.yaml:9]: new feature should raise minor version.

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-node-exporter/values.yaml (hunk 1 above)
Comment (TomasKohout) [reply charts/prometheus-node-exporter/values.yaml]: Srry, format on safe change. :slightly_smiling_face:

**Review by monotek — APPROVED** [review]
